types/react-hyperscript/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
```

Comment:
With this line, tsc 2.5 raises an error:
> tsconfig.json(10,9): error TS5023: Unknown compiler option 'strictFunctionTypes'.

Without this line, `npm run lint react-hyperscript` raises an error:
> Error: Expected `"strictFunctionTypes": true` or `"strictFunctionTypes": false`.

After some googling, I found [a comment that says that DefinitelyTyped's test runner will ignore the latter](https://github.com/Microsoft/TypeScript/issues/19294#issuecomment-337646626)
So, I decided to remain this line unchanged.

Reply:
Yeah, it's fine, we'll ignore the error for older TS versions.